### DISCUSSION TITLE
Customizing Top Bar

### DISCUSSION THREAD

**Discussion by hrija** [Aylur/ags#136] [General]
Hi guys,

Sorry if this is the wrong place for this, but I recently copied the bar example, since it seemed like a good base to start on, and am in the process of editing and changing everything up. 

I was just wondering how to create the floating bar effect I've seen some people have, I tried adding stuff like padding-top and border-radius in the css for the bar, but those don't seem to be working. 

Any suggestions?

Btw I'm pretty new to css and js so sorry if the answer is obvious!

Also is there a discord server?

**Comment by Aylur** [comment]
there are multiple ways, you can use the margin property on the [Window](https://github.com/Aylur/ags/wiki/Basic-Widgets#window) widget
or you can use css tricks like this where you apply margin to the inner box in the window
```css
window > box {
    margin: 12px;
}
```

>Also is there a discord server?

yes, it is linked at the bottom of the [wiki home page](https://github.com/Aylur/ags/wiki)
